egui/src/context.rs

```diff
@@ -574,6 +575,39 @@ impl Context {
         }
     }
 
+    /// request repaint after the specified duration elapses in the case of no new input
```

Comment:
I guess i like writing documentation xD

Reply:
I guess you don't like capitalization :P